packages/ERTP/src/collections.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
+
+/**
+ * TODO: move to https://github.com/Agoric/readonly-collections-shim ?
+ *
+ * @returns {Set<T>}
+ * @template T
+ */
+export function makeFlexSet() {
+  return new Set();
```

Comment:
We should preserve the parameters for all constructors we wrap this way, except when there's a reason not to.

```suggestion
export function makeFlexSet(iterable = undefined) {
  return new Set(iterable);
```

Reply:
I was hesitant to add untested code, but I agree this is better.